Makefile

```diff
@@ -20,6 +20,12 @@ GO_VERSION = 1.20
 ENVTEST_K8S_VERSION = 1.28
 # See https://github.com/slintes/sort-imports/releases for the last version
 SORT_IMPORTS_VERSION = v0.2.1
+# OCP Version: for OKD bundle community
+OCP_VERSION ?= 4.12
```

Comment:
Do we need this here? It's set by CI, not?

Reply:
I remember a discussion where we ended up wanting at least a default version set :shrug:

Reply:
btw, some other operators have a bug here (SNR and MDR, IIRC), because they assign the value without the `?`.

Reply:
ok...

Reply:
aaand I remembered incorrectly... sorry :sweat: 
_at the time of the discussion_ the variable was useful because we didn't have this CI

https://github.com/medik8s/node-healthcheck-operator/pull/300#discussion_r1530151953